modules/RTC/TraceablePeerConnection.js

```diff
@@ -3104,8 +3094,7 @@ TraceablePeerConnection.prototype.generateNewStreamSSRCInfo = function(track) {
     }
 
     // Configure simulcast for camera tracks and desktop tracks that need simulcast.
-    if (this.isSimulcastOn()
-        && (track.getVideoType() === VideoType.CAMERA || !this.isSharingLowFpsScreen())) {
```

Comment:
Why do we remove the `!this.isSharingLowFpsScreen()`?

Also probably the comment needs an update and also the braces are redundant.

Reply:
This is part of plan-b flow and not executed anymore. I will do another PR for cleaning up plan-b stuff.

Reply:
If you prefer, I can leave this in for now.